### DISCUSSION TITLE
Prevent Command From Being Recorded in History

### DISCUSSION THREAD

**Discussion by anuade** [cmderdev/cmder#2503] [Q&A] [answered]
Sometimes it is useful to exclude a command from being recorded in history. Is there any straightforward way to do this apart from, say, running a batch file?  For example in bash, a space before the command would prevent its being recorded. Thank you.

**Comment by chrisant996** [comment, marked as answer]
By default, a space prevents a command from being recorded in history.

There is a setting which controls that. Refer to the docs for more info.
